src/oge/helpers.py

```diff
@@ -366,7 +366,7 @@ def add_plant_operating_and_retirement_dates(df: pd.DataFrame) -> pd.DataFrame:
             'plant_retirement_date' column.
     """
     generator_dates = load_data.load_pudl_table(
-        "denorm_generators_eia",
+        "out_eia__yearly_generators",
```

Comment:
In `oge.data_cleaning`, it is `_out_eia__yearly_generators`. See this [comment](https://github.com/singularity-energy/open-grid-emissions/pull/369/files#r1648210955).

Reply:
See above response. I use the _ version in the allocation pipeline because that is what is used by pudl there, but here I want to use the more general version of this table.